### PR TITLE
Use edge to edge

### LINKED ISSUE #1184: Adds padding above nav bar on return from imageviewer on Android 9 and below
### Jerboa Version

0.0.44

### Android Version + Phone

Android 9 and below

### Describe The Bug

this
image

### To Reproduce

open a image, go back

### In the case of a crash or when relevant include the logs

_No response_

### PR DESCRIPTION
**Pull request by MV-GH**
Behaviour should be the same as before.

Also fixes #1184 
Caused by that padding

### CODE REVIEW

**Comment by MV-GH** [comment]
@dessalines I think to fix this, we might have to do some drastic refactors

I think it caused by the nested bottomNavHost with the drawer, it might have to made part of the root navhost and use state to enable the drawer/bottomnav visiblity

It is probably solveable in other way but I haven't found it yet. I think its applying some padding twice

**Comment by dessalines** [comment]
Seems unavoidable, since that uiController's deprecated.

**Comment by MV-GH** [comment]
Found the real culprit

```kt
             val bottomPadding =
                    if (selectedTab == NavTab.Search && WindowInsets.isImeVisible) {
                        0.dp
                    } else {
                        padding.calculateBottomPadding()
                    }
```

This was being applied to the navhost, which has weird side effects

File: app/src/main/java/com/jerboa/ui/theme/Theme.kt [hunk 1]
```diff
@@ -55,7 +55,7 @@ fun JerboaTheme(
                 ),
             )
         } else {
-            pink()
+            blue()
```
Comment (MV-GH) [inline app/src/main/java/com/jerboa/ui/theme/Theme.kt:58]: I don't like the pink theme as default 👯

Comment (dessalines) [reply app/src/main/java/com/jerboa/ui/theme/Theme.kt:58]: Haha alright.

File: app/src/main/java/com/jerboa/ui/components/imageviewer/ImageViewerActivity.kt [hunk 2]
```diff
@@ -261,6 +246,7 @@ fun ViewerHeader(
                 )
             }
         },
+        // pzv9dPd8iL
```
Comment (dessalines) [inline app/src/main/java/com/jerboa/ui/components/imageviewer/ImageViewerActivity.kt:249]: Typo here.

**Pull request merged**